### PR TITLE
Solving RESTWS-713 - Add free-text cause of death to Person resource

### PR DESCRIPTION
**Pull request by fruether**
Adding new attribute to set of return values
Create one simple test case to make sure everything is working
https://issues.openmrs.org/browse/RESTWS-713

Adapted requested changes of: https://github.com/openmrs/openmrs-module-webservices.rest/pull/337

### CODE REVIEW

File: omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		description.addProperty("causeOfDeathNonCoded");
+		return description;
+	}
+	
+	/**
+	 * @throws org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException
+	 * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getUpdatableProperties()
+	 */
+	@Override
+	public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoesNotSupportOperationException {
```
Comment (wluyima) [inline omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: By default getUpdatableProperties() returns the same properties as getCreatableProperties(), it means you don't have to implement this if the properties are going to be the same.

Comment (wluyima) [reply omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: Did you see this comment?

Comment (fruether) [reply omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: I fixed it now

File: omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_11.PersonResource1_11;
+
+/**
+ * {@link Resource} for Person, supporting standard CRUD operations
+ */
+@Resource(name = RestConstants.VERSION_1 + "/person", supportedClass = Person.class, supportedOpenmrsVersions = {
+        "2.2.*" })
+public class PersonResource2_2 extends PersonResource1_11 {
```
[the remark this answers is not in the document]
Comment (fruether) [reply omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: Done

[the remark this answers is not in the document]
Comment (wluyima) [reply omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: Thanks!

**Pull request merged**